runtime/doc/builtin.txt

```diff
@@ -4261,6 +4263,36 @@ getreginfo([{regname}])					*getreginfo()*
 		Can also be used as a |method|: >
 			GetRegname()->getreginfo()
 
+getregion({pos1}, {pos2}, {type})			*getregion()*
+		Returns the list of strings from {pos1} to {pos2}.
+		For possible values of {pos1} and {pos2} see |line()|.
+		The {type} is region type:
+			"v" for |characterwise| mode
+			"V" for |linewise| mode
+			"<CTRL-V>" for |blockwise-visual| mode
+		You can get current selection type by |visualmode()|.
+		This function uses the line and column number from the
+		specified position.
+		It is useful to get text starting and ending in different
+		columns, such as |characterwise-visual| selection.
+
+		Note that:
+		- Order of {pos1} and {pos2} doesn't matter, it will always
+		  return content from the upper left position to the lower
+		  right position.
+		- If 'virtualedit' is enabled and selection is past the end of
+		  line, result lines are filled.
+		- If selection starts or ends in the middle of a multibyte
+		  character, it is not included but its selected part is
+		  substituted with spaces.
+
+		Examples: >
+			:xnoremap <CR>
+			\ <Cmd>echo getregion('v', '.', visualmode())<CR>
```

Comment:
`visualmode()` returns the type of the last Visual mode, while a `<Cmd>` mapping remains in the current Visual mode, which should be obtained using `mode()`, so this mapping should either using `mode()` instead of `visualmode()` or use `:` instead of `<Cmd>`.